app/src/main/java/com/github/h3lp3rs/h3lp/messaging/ChatActivity.kt

```diff
@@ -91,27 +87,18 @@ class ChatActivity : AppCompatActivity() {
         fun onChildRemoved(key : String) {
             // If the key is the conversation Id, that means that the user deleted the current
```

Comment:
Since the check on key == conversationId now occurs Conversation.kt, you could switch this comment over to there